api.js

```diff
@@ -137,6 +144,11 @@ Api.prototype.run = function () {
 				return Promise.reject(new AvaError('Couldn\'t find any files to test'));
 			}
 
+			var cache = self.options.cache !== false;
+			var cacheDir = (cache && findCacheDir({name: 'ava', files: files})) || uniqueTempDir();
```

Comment:
Maybe split this into multiple lines? Does not look clean when everything is on one.